### PR TITLE
Mark all notifications as read

### PR DESCRIPTION
**Pull request by Flowtter**
Add button 'mark all as read' in the notification modal to clear all the notifications.
image

### CODE REVIEW

File: core/modules/notifications/client/notifications.js [hunk 1]
```diff
@@ -81,6 +83,14 @@ Template.notification.events({
   },
 });
 
+Template.notifications.events({
+  'click .js-notification-mark-all-as-read'(event) {
+    event.preventDefault();
+    markAllNotificationsAsRead();
+    closeModal('notifications');
```
Comment (ramnes) [inline core/modules/notifications/client/notifications.js]: usually not a good UX practice to launch two actions on a single click

File: core/modules/notifications/client/notifications.hbs.html [hunk 2]
```diff
@@ -32,6 +32,9 @@
   {{#modal id="displayNotificationsPanel" title="Notifications"}}
     <div class="header">
       <h2 class="h2">Notifications</h2>
+      <h4 class="h4">
+        <button type="button" class="mark-as-read js-notification-mark-all-as-read">Mark all as read</button>
+      </h4>
```
Comment (ramnes) [inline core/modules/notifications/client/notifications.hbs.html]: Maybe an icon + hover à la Gmail would be nicer?

image

**Review by ramnes — APPROVED** [review]
LGTM

File: core/modules/notifications/client/notifications.hbs.html (hunk 2 above)
Comment (Flowtter) [reply core/modules/notifications/client/notifications.hbs.html]: image
Added in: https://github.com/Refty/lemverse/pull/26/commits/74b730f9294d3fca8bbfca203cbe84503c1b7229

File: core/modules/notifications/client/notifications.js (hunk 1 above)
Comment (Flowtter) [reply core/modules/notifications/client/notifications.js]: fixed in https://github.com/Refty/lemverse/pull/26/commits/fe2c8ac15fd9cde9765d77b3cab9f79ad8d9670d

File: core/modules/notifications/client/notifications.hbs.html (hunk 2 above)
Comment (ramnes) [reply core/modules/notifications/client/notifications.hbs.html]: nice

**Pull request merged**